runtime/lua/vim/lsp/util.lua

```diff
@@ -203,6 +203,11 @@ function M.text_document_completion_list_to_complete_items(result, prefix)
       icase = 1,
       dup = 1,
       empty = 1,
+      user_data = M.json_encode({
+        lsp = {
```

Comment:
I don't know the nvim's `user_data` key's policy.
Currently, I choose `lsp` key but we can choose other candidates, I think.

For example,

- { nvim = { lsp = { ... } }
- { nvim_lsp = { ... } }

Reply:
@Shougo would know

Reply:
I think we should use key that has not possibility to conflict.
So I will change key to `{ nvim = { lsp = { ... } } }` instead of `{ lsp = { ... } }`

Reply:
ok